entrypoints/components/ConnectionDetails.tsx

```diff
@@ -0,0 +1,54 @@
+import { Database, MapPin } from '@phosphor-icons/react'
+import { Dropdown, DropdownSection } from './Dropdown'
+
+interface ConnectionDetailsProps {
+  selectedLocation: string
+  dropdownSection: DropdownSection[]
```

Comment:
if the prop dropdownSection is an array of DropdownSection maybe not has to have another name, like dropdownSections?